src/ui/preferences/GeneralSettings.qml

```diff
@@ -1078,13 +1099,34 @@ Rectangle {
                                     onAcceptedForLoad:  _userBrandImageOutdoor.rawValue = "file:///" + file
                                 }
                             }
+
+
+                            QGCLabel {
+                                text:           qsTr("Outdoor Image Margin")
+                                visible:        userBrandImageOutdoorMargin.visible
+                            }
+                            SpinBox {
```

Comment:
We don't have a QGC version of a SpinBox. You can't use the Qt controls directly since they don't work with the QGC pallete selection/theming. This will need to just a a QGCTextField.

Reply:
Can we make a SpinBox for QGC?

Reply:
> Can we make a SpinBox for QGC?

It depends on who "we" is :) I don't have time to do it myself right now. But I assume it should be doable. If you do, start with controls 2 version.

Reply:
Ok, I'll try to do it. Any documentation to helping me in this journey :rofl: ?

Reply:
Not really. Other than looking at how the other QGCFoo controls do it. They need to respect the palette colors and font sizing. I would change this to a QGCTextField to complete this pull. And then try doing the spin box in a separate pull. It certainly would be nice to have a spin box as a usable QGC control.

Reply:
Ok thanks :)